lib/check/check.py

```diff
@@ -145,22 +149,30 @@ def import_check(check_path: str) -> ModuleType:
     return lib
 
 
-def set_output_options(quiet):
+def set_output_options(quiet: bool, output_modes: list):
```

Comment:
If this function does not have test, please create a new story to address it in the future.